pandas/tests/reshape/merge/test_join.py

```diff
@@ -415,7 +416,7 @@ def test_join_inner_multiindex(self, lexsorted_two_level_string_multiindex):
         expected = expected.drop(["first", "second"], axis=1)
         expected.index = joined.index
 
-        assert joined.index.is_monotonic_increasing
+        # assert joined.index.is_monotonic_increasing
```

Comment:
Can you revert this?

Reply:
this would fail.. since `to_join` is being joined on the index.. the final df would contain that index

this behaviour is mentioned in the docs for `merge`.. but since join also uses most of the same stuff.. I think they should be consistent?!

Reply:
Hmm not sure I follow - what does this produce now? Why does the monotonicity of the index change?